.github/workflows/deploy-production-subgraph.yaml

```diff
@@ -0,0 +1,35 @@
+name: Production Subgraph (Mainnet)
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Install commands
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          registry-url: https://registry.npmjs.org
+      - name: yarn add @graphprotocol/graph-cli
```

Comment:
It's actually better to not have this step, since it makes it so that the graph-cli versions always match what's being used in the project.

[Check these changes](https://github.com/graphprotocol/graph-network-subgraph/blob/master/.github/workflows/deploy-production-subgraph.yaml#L16-L27) in the core network subgraph for reference